Error: Cannot find module 'add-eventlistener-with-options' from ...

Bumped from v1.25.0 to 1.25.3 in my https://github.com/binarykitchen/videomail-client package and now unit tests fail.

Here a stack trace, not very helpful I am afraid:

```
[17:02:44] Starting 'test'...
Error: Cannot find module 'add-eventlistener-with-options' from '/Users/michael-heuberger/code/videomail-client/src/wrappers/visuals'
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^

Error: javascript required
    at Stream.<anonymous> (/Users/michael-heuberger/code/videomail-client/node_modules/browser-run/index.js:35:34)
    at _end (/Users/michael-heuberger/code/videomail-client/node_modules/through/index.js:65:9)
    at Stream.stream.end (/Users/michael-heuberger/code/videomail-client/node_modules/through/index.js:74:5)
    at Stream.method [as end] (/Users/michael-heuberger/code/videomail-client/node_modules/duplexer/index.js:47:39)
    at Stream.<anonymous> (/Users/michael-heuberger/code/videomail-client/node_modules/throughout/index.js:7:25)
    at _end (/Users/michael-heuberger/code/videomail-client/node_modules/through/index.js:65:9)
    at Stream.stream.end (/Users/michael-heuberger/code/videomail-client/node_modules/through/index.js:74:5)
    at Stream.onend (internal/streams/legacy.js:44:10)
    at emitNone (events.js:111:20)
    at Stream.emit (events.js:208:7)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

The interesting part is probably

```
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
```

and

```
Error: javascript required
```

--- a/.github/main.workflow
+++ b/.github/main.workflow
@@ -1,9 +1,25 @@
-workflow "publish on release" {
+workflow "build and test" {
   on = "push"
+  resolves = ["build", "test"]
+}
+
+action "test" {
+  uses = "actions/npm@master"
+  args = "test"
+}
+
+action "build" {
+  uses = "actions/npm@master"
+  args = "build"
+}
+
+workflow "publish on release" {
+  on = "release"
   resolves = ["publish"]
 }
 
 action "publish" {
+  needs = "build"
   uses = "actions/npm@master"
   args = "publish"
   secrets = ["NPM_AUTH_TOKEN"]

--- a/.github/main.workflow
+++ b/.github/main.workflow
@@ -1,16 +1,23 @@
 workflow "build and test" {
   on = "push"
-  resolves = ["build", "test"]
+  resolves = ["build", "cover"]
 }
 
-action "test" {
+action "install" {
   uses = "actions/npm@master"
-  args = "test"
+  args = "install"
 }
 
 action "build" {
+  needs = "install"
   uses = "actions/npm@master"
   args = "build"
+}
+
+action "cover" {
+  needs = "install"
+  uses = "actions/npm@master"
+  args = "cover"
 }
 
 workflow "publish on release" {

--- a/.github/main.workflow
+++ b/.github/main.workflow
@@ -17,7 +17,7 @@ action "build" {
 action "cover" {
   needs = "install"
   uses = "actions/npm@master"
-  args = "cover"
+  args = "run cover"
 }
 
 workflow "publish on release" {

--- a/.npmignore
+++ b/.npmignore
@@ -6,6 +6,11 @@ dist/
 assets/
 coverage/
 .nyc_output
+.github
+
+# Ignore files
+CODE_OF_CONDUCT.md
+contributing.json
 
 # Ignore build and environment specific files
 yarn.lock

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "add-eventlistener-with-options",
-  "version": "1.25.3",
+  "version": "1.25.4",
   "description": "A utility function to check if EventTarget.addEventListener supports adding passive events",
   "main": "index.js",
   "scripts": {
@@ -44,6 +44,7 @@
     "mocha-jsdom": "1.1.0",
     "nyc": "10.0.0",
     "rimraf": "2.5.4",
+    "sinon": "7.3.2",
     "webpack": "1.13.2"
   },
   "nyc": {

--- a/src/index.test.js
+++ b/src/index.test.js
@@ -1,0 +1,21 @@
+import { expect } from 'chai';
+import jsdom from 'mocha-jsdom';
+import sinon from 'sinon';
+import addEventListenerWithOptions from "./index";
+
+describe("addEventListenerWithOptions", () => {
+    jsdom();
+
+    it("should be able to attach event listener successfully", () => {
+        const target = document.createElement("button");
+        target.id = "target-id";
+        target.textContent = "Hello World!";
+        const mockListener = sinon.spy();
+        addEventListenerWithOptions(target, "click", mockListener);
+        const evt = document.createEvent("HTMLEvents");
+        evt.initEvent("click", false, true);
+        target.dispatchEvent(evt)
+
+        expect(mockListener.callCount).to.equal(1);
+    });
+});

